src/js/front-end/setup/deck-constructor/find-type.js

```diff
@@ -117,7 +129,11 @@ const SVEra = {
         125: "Trainer",
         150: "Pokémon",
         151: "Trainer",
-        163: "Pokémon",
+        167: "Pokémon",
```

Comment:
since these type definitions are only based off of limitless URLs, i feel like we shouldn't update this until limitless actually uploads the cards first.

right now, limitless's SVP database only goes up to 165: https://limitlesstcg.com/cards/SVP

Also, since you made this commit, I see bulbapedia has already added more cards, e.g., 167 - 173 are the new eeveelutions promos.

So, I think we should only update this as Limitless updates their stuff.

IMO this whole process should be reworked so we don't have to manually update the types, but that is another rabbithole. Open to suggestions.

Reply:
I removed all references to cards past 165. However there are still cards in there that are not on Limitless, namely 85 (Pikachu with Grey Felt Hat) and 102 (Oddish)